From ad48917cc283184a776dc570c41de9808cdbde5b Mon Sep 17 00:00:00 2001
From: el1191 <el1191@woyd.ru>
Date: Sun, 23 Aug 2026 18:18:35 +0300
Subject: [PATCH 6/6] Fix version

---
 .gitignore | 3 +++
 VERSION    | 1 -
 2 files changed, 3 insertions(+), 1 deletion(-)
 delete mode 100644 VERSION

diff --git a/.gitignore b/.gitignore
index d675c48..95f2620 100644
--- a/.gitignore
+++ b/.gitignore
@@ -38,3 +38,6 @@ cmd/argocd-server/argocd-server
 # ignore generated `.argocd-helm-dep-up` marker file; this should not be committed to git
 reposerver/repository/testdata/**/.argocd-helm-dep-up
 .claude
+
+# Replace on build step
+VERSION
diff --git a/VERSION b/VERSION
deleted file mode 100644
index d5c0c99..0000000
--- a/VERSION
+++ /dev/null
@@ -1 +0,0 @@
-3.5.2
-- 
2.54.0

